.project

```diff
@@ -0,0 +1 @@
+/Users/brianpollack/Coding/scikit-hep
```

Comment:
Can you remove this file and add a line in .gitignore so that such files are hereafter ignored :-)?

Reply:
whoops, of course